ui/ducks/confirm/confirm.ts

```diff
@@ -1,6 +1,6 @@
 type ActionType = {
   type: string;
-  currentConfirmation?: Record<string, unknown> | undefined;
+  propsToUpdate?: Record<string, unknown> | undefined;
 };
```

Comment:
I would suggest not changing currentConfirmation in state as all of confirmation page components depend on it. If needed additional parameters can be added to the state outside `currentConfirmation`.

Reply:
Hey @digiwand : a change to current confirmation object can trigger re-rendering across confirmation pages, let's sync to discuss if this can be avoided.

Reply:
updated state and redux to support 
```
confirm: {
  currentConfirmation: { ... },
  isScrollToBottomNeeded: true,
}
```
rather than 
```
confirm: {
  currentConfirmation: { 
    ...props
    isScrollToBottomNeeded: true,
  }
}
```